spec/factories/clearance.rb

```diff
@@ -4,7 +4,16 @@
   end
 
   factory :user do
+    name "Jon Smith"
     email
     password "password"
+
+    trait :admin do
```

Comment:
@paulcsmith, Any advice you have here would be great? I may have incorrectly interpreted nested factories as building out traits.